roles/setup_dbt2/tasks/install_packages.yml

```diff
@@ -0,0 +1,17 @@
+---
+
+- name: Set the os variable
+  set_fact:
+    os: "{{ ansible_distribution | replace('RedHat', 'RHEL') }}{{ ansible_distribution_major_version }}"
```

Comment:
I'd suggestion to not change the distribution name, please work with `RedHat`. I know that there are still some areas where we work with `RHEL`, we need to fix that in the future.

Reply:
Ok,fixed.